MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -45,6 +45,20 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
      */
     @objc public var abbreviationPriority: Int = NSNotFound
     
+    /**
+     An array indicating which directions you can go from a lane (left, right, or straight).
+     
+     If the value is `[.left", .straight]`, the driver can go left or straight from that lane. This is only set when the `component` is a `lane`.
+     */
+    @objc public var indications: LaneIndication
+    
+    /**
+     The boolean that indicates whether the component is a lane and can be used to complete the upcoming maneuver.
+     
+     If multiple lanes are active, then they can all be used to complete the upcoming maneuver. This value is set to `false` by default.
+     */
+    @objc public var isLaneActive: Bool = false
```

Comment:
Rename this property to `isActiveLane`, or `isActive` if you end up refactoring the class. Also refine the getter and setter names in Objective-C: `-activeLane` should be the getter, and `-setActiveLane:` should be the setter.